app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,44 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
```

Comment:
Please make sure that a user can see the foods created by other users on the `foods#index` page.

Reply:
@SajjadAhmad14 ,  according to the instructions, i am only meant to display the foods by the logged in user. Therefore I do not understand this comment. Please explain further.

Reply:
Thanks @KDlamini 🤗 for helpiing us resolve this.